controller/array_action/array_mediator_ds8k.py

```diff
@@ -215,33 +223,20 @@ def _generate_volume_response(self, api_volume):
             array_address=self.service_address,
             copy_source_id=self._get_copy_source_id(api_volume=api_volume),
             pool_name=api_volume.pool,
-            array_type=self.array_type
+            array_type=self.array_type,
+            space_efficiency=api_volume.tp
```

Comment:
wouldn't it make more sense to assign here the cross-storage server equivalent of the space efficiency?

Reply:
done